CRM/Core/Invoke.php

```diff
@@ -55,7 +55,7 @@ public static function _invoke($args) {
       return NULL;
     }
     // CRM-15901: Turn off PHP errors display for all ajax calls
-    if (CRM_Utils_Array::value(1, $args) == 'ajax' || CRM_Utils_Array::value('snippet', $_REQUEST)) {
+    if (CRM_Utils_Array::value(1, $args) == 'ajax' || !empty($_REQUEST['snippet'])) {
       ini_set('display_errors', 0);
     }
```

Comment:
The replacement is fine but this has bugged me for a while and I wonder if there's a way to only do this when in production. Devs should have errors mess up their snippets. 😈